src/webui/www/.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "env": {
+        "browser": true,
+        "es2020": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-undef": "off",
```

Comment:
Why do this checks have to be off? Does the current code require many changes to get these working? They seem useful.

https://eslint.org/docs/rules/no-undef
https://eslint.org/docs/rules/no-unused-vars

Reply:
eslint can't correctly resolve the variables, so I disabled them for now and hoping later someone would find a way to make it work.

Reply:
@Chocobo1 OK. Is it possible to add that explanation to that file? If not, I would suggest adding it to the commit message.

Reply:
>Is it possible to add that explanation to that file? 

It can't.

>I would suggest adding it to the commit message.

Done.